intermediate_notebooks/E2E/taxi/NYCTaxi-E2E.ipynb

```diff
@@ -316,13 +450,235 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 18,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 788 ms, sys: 24 ms, total: 812 ms\n",
```

Comment:
Is there a reason we're timing this cell now?

Reply:
Was done for debugging.